bundler/lib/bundler/lockfile_parser.rb

```diff
@@ -177,11 +188,13 @@ def parse_dependency(line)
     def parse_spec(line)
       return unless line =~ NAME_VERSION
       spaces = $1
-      name = $2
+      name = -$2
       version = $3
-      platform = $4
 
       if spaces.size == 4
+        # no platform for a dep line
```

Comment:
This comment is confusing because it's saying "no platform" right where you set platform. Maybe "only load platform for non-dependency line"